app/analysis/edit/controller.js

```diff
@@ -127,6 +127,18 @@ export default Controller.extend(AnalysisEditQueryParams.Mixin, {
         'intersection.lastSuccessful.value.model'
       ).serialize()
 
+      params.id.split(',').forEach(async id => {
+        if (id) {
```

Comment:
Maybe use `filter` instead of `if` inside `forEach`?

Reply:
Added ```without('')``` the problem was for an empty string it would put an empty string into the array.

Reply:
Awesome, didn't know about `without`!